diff -urN ppplicznik-1.0.0-org/src/ekran.cpp ppplicznik-1.0.0/src/ekran.cpp
--- ppplicznik-1.0.0-org/src/ekran.cpp	Tue Oct 30 22:11:21 2001
+++ ppplicznik-1.0.0/src/ekran.cpp	Sun Nov  4 12:54:56 2001
@@ -19,7 +19,7 @@
 
 #pragma implementation
 
-#include <ncurses.h>
+#include <ncurses/ncurses.h>
 #include <unistd.h>
 
 #include <string>
diff -urN ppplicznik-1.0.0-org/src/ekran.h ppplicznik-1.0.0/src/ekran.h
--- ppplicznik-1.0.0-org/src/ekran.h	Mon Oct 29 18:58:54 2001
+++ ppplicznik-1.0.0/src/ekran.h	Sun Nov  4 12:55:03 2001
@@ -22,7 +22,7 @@
 
 #pragma interface
 
-#include <ncurses.h>
+#include <ncurses/ncurses.h>
 #include <unistd.h>
 
 class ekran
diff -urN ppplicznik-1.0.0-org/src/main.cpp ppplicznik-1.0.0/src/main.cpp
--- ppplicznik-1.0.0-org/src/main.cpp	Fri Nov  2 08:46:35 2001
+++ ppplicznik-1.0.0/src/main.cpp	Sun Nov  4 12:55:08 2001
@@ -24,7 +24,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/wait.h>
-#include <ncurses.h>
+#include <ncurses/ncurses.h>
 
 #include <fstream>
 #include <iostream>
